### PR TITLE
Update format type field to be called 'processing type'

### PR DESCRIPTION
**Pull request by gkillough**

### CODE REVIEW

File: alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/ProviderDescriptor.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
     public static final String KEY_PROVIDER_CONFIG_NAME = KEY_COMMON_PROVIDER_PREFIX + "config.name";
     public static final String KEY_PROVIDER_CONFIG_ENABLED = KEY_COMMON_PROVIDER_PREFIX + "config.enabled";
     public static final String LABEL_PROVIDER_CONFIG_ENABLED = "Enabled";
-    public static final String LABEL_PROVIDER_CONFIG_NAME = "Configuration Name";
+    public static final String LABEL_PROVIDER_CONFIG_NAME = "Provider Configuration";
```
Comment (bamandel) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/ProviderDescriptor.java]: I think having name in the label here is still reasonable

File: alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ConfigurationJobModel.java [hunk 2]
```diff
@@ -84,8 +84,8 @@ public FrequencyType getFrequencyType() {
         return getFieldAccessor().getEnum(ChannelDistributionUIConfig.KEY_FREQUENCY, FrequencyType.class).orElse(null);
     }
 
-    public FormatType getFormatType() {
-        return getFieldAccessor().getEnum(ProviderDistributionUIConfig.KEY_FORMAT_TYPE, FormatType.class).orElse(null);
+    public ProcessingType getFormatType() {
```
Comment (bamandel) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ConfigurationJobModel.java]: Should the method name change here?

Comment (jamesrichard91) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ConfigurationJobModel.java]: Should we change the method name as well?

File: alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ConfigurationJobModel.java [hunk 3]
```diff
@@ -84,8 +84,8 @@ public FrequencyType getFrequencyType() {
...
+        return getFieldAccessor().getEnum(ProviderDistributionUIConfig.KEY_FORMAT_TYPE, ProcessingType.class).orElse(null);
```
Comment (jamesrichard91) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ConfigurationJobModel.java]: should we change the KEY_FORMAT_TYPE to KEY_PROCESSING_TYPE?

File: alert-common/src/main/java/com/synopsys/integration/alert/common/provider/ProviderContent.java [hunk 4]
```diff
@@ -25,13 +25,13 @@
 import java.util.Set;
 
 import com.google.common.collect.ImmutableSet;
-import com.synopsys.integration.alert.common.enumeration.FormatType;
+import com.synopsys.integration.alert.common.enumeration.ProcessingType;
 
 public abstract class ProviderContent {
     private final Set<ProviderNotificationType> supportedNotificationTypes;
-    private final Set<FormatType> supportedContentFormats;
+    private final Set<ProcessingType> supportedContentFormats;
```
Comment (jamesrichard91) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/provider/ProviderContent.java]: should we change the field to supportedProcessingTypes?

File: alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/formatter/MessageContentFormatter.java [hunk 5]
```diff
@@ -27,19 +27,19 @@
...
 
-import com.synopsys.integration.alert.common.enumeration.FormatType;
+import com.synopsys.integration.alert.common.enumeration.ProcessingType;
 import com.synopsys.integration.alert.common.message.model.ContentKey;
 import com.synopsys.integration.alert.common.message.model.MessageContentGroup;
 import com.synopsys.integration.alert.common.message.model.ProviderMessageContent;
 
 public abstract class MessageContentFormatter {
-    private final FormatType formatType;
+    private final ProcessingType formatType;
```
Comment (jamesrichard91) [inline alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/formatter/MessageContentFormatter.java]: Should we change the field name and associated get method?

File: src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckContent.java [hunk 6]
```diff
@@ -48,7 +48,7 @@
         NotificationType.PROJECT_VERSION
     );
 
-    private static final EnumSet<FormatType> SUPPORTED_CONTENT_FORMATS = EnumSet.of(FormatType.DEFAULT, FormatType.DIGEST, FormatType.SUMMARY);
+    private static final EnumSet<ProcessingType> SUPPORTED_CONTENT_FORMATS = EnumSet.of(ProcessingType.DEFAULT, ProcessingType.DIGEST, ProcessingType.SUMMARY);
```
Comment (jamesrichard91) [inline src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckContent.java]: Should we change the name of this stack field?

**Pull request merged**